### ISSUE TITLE
Can't install golangci-lint normally on Ubuntu 20.04

### ISSUE THREAD

**Issue by Memtwo** [golangci/golangci-lint#1516]
Thank you for creating the issue!

- [x] Yes, I'm using a binary release within 2 latest major releases. Only such installations are supported.
- [x] Yes, I've searched similar issues on GitHub and didn't find any.
- [x] Yes, I've included all information below (version, config, etc).

Please include the following information:

<details><summary>Version of golangci-lint</summary>

```console
$ golangci-lint --version
# paste output here
```
golangci-lint：command not found
</details>

<details><summary>Config file</summary>

```console
$ cat .golangci.yml
# paste output here
```
no cause i can't install it
</details>

<details><summary>Go environment</summary>

```console
$ go version && go env
# paste output here
```
go version go1.13.8 linux/amd64
GO111MODULE=""
GOARCH="amd64"
GOBIN=""
GOCACHE="/root/.cache/go-build"
GOENV="/root/.config/go/env"
GOEXE=""
GOFLAGS=""
GOHOSTARCH="amd64"
GOHOSTOS="linux"
GONOPROXY=""
GONOSUMDB=""
GOOS="linux"
GOPATH="/root/go"
GOPRIVATE=""
GOPROXY="https://proxy.golang.org,direct"
GOROOT="/usr/lib/go-1.13"
GOSUMDB="sum.golang.org"
GOTMPDIR=""
GOTOOLDIR="/usr/lib/go-1.13/pkg/tool/linux_amd64"
GCCGO="gccgo"
AR="ar"
CC="gcc"
CXX="g++"
CGO_ENABLED="1"
GOMOD=""
CGO_CFLAGS="-g -O2"
CGO_CPPFLAGS=""
CGO_CXXFLAGS="-g -O2"
CGO_FFLAGS="-g -O2"
CGO_LDFLAGS="-g -O2"
PKG_CONFIG="pkg-config"
GOGCCFLAGS="-fPIC -m64 -pthread -fmessage-length=0 -fdebug-prefix-map=/tmp/go-build340028246=/tmp/go-build -gno-record-gcc-switches"

</details>

<details><summary>Verbose output of running</summary>

```console
$ golangci-lint cache clean
$ golangci-lint run -v
# paste output here
```
golangci-lint：command not found
</details>

**Comment by sayboras** [comment]
hard to know what the issue, can you at least add the steps that you followed for installation ?

**Comment by Memtwo** [comment]
step:
root@fjd-virtual-machine:/home/fjd# curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(go env GOPATH)/bin v1.32.2
golangci/golangci-lint info checking GitHub for tag 'v1.32.2'
golangci/golangci-lint info found version: 1.32.2 for v1.32.2/linux/amd64
golangci/golangci-lint info installed /root/go/bin/golangci-lint
root@fjd-virtual-machine:/home/fjd# golangci-lint --version
golangci-lint：command not found

**Comment by sayboras** [comment]
can you check if $(go env GOPATH)/bin in your $PATH? This is most likely the case.

**Comment by sayboras** [comment]
Closed this issue, feel free to reopen if required.

**Comment by Domarx** [comment]
Hello.

It seems that I have the same issue.

I wanted to update the version so I deleted the existing binary `golangci-lint` from `./go/bin`.
Then I installed the new version:
```
curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(go env GOPATH)/bin v1.40.1

golangci/golangci-lint info checking GitHub for tag 'v1.40.1'
golangci/golangci-lint info found version: 1.40.1 for v1.40.1/linux/amd64
golangci/golangci-lint info installed /home/domarx/go/bin/golangci-lint
```

Then I went to the directory that contains the binary and tried to start the program `./golangci-lint ` but I got this:
`ERRO Failed to init hash salt: failed to calculate config salt: failed to json marshal config linter settings: json: unsupported type: map[interface {}]interface {}`

Tried this multiple times but to no avail. 
But then I downloaded source code, and tried to run the program from the source code directory it worked.
